lib/gelf/logger.rb

```diff
@@ -5,7 +5,7 @@ module LoggerCompatibility
     attr_accessor :formatter
 
     # Use it like Logger#add... or better not to use at all.
-    def add(level, message = nil, progname = nil)
+    def add(level, message = nil, progname = nil, &block)
```

Comment:
Yes, they removed it but since there may very well still be a few users of this library who are stuck on Ruby 1.9, we can keep the block argument for the moment. I'd just prefer to actually use it (like `message ||= block.call unless block.nil?`) so that it isn't a dead argument.